### DISCUSSION TITLE
CMake Warning

### DISCUSSION THREAD

**Discussion by Pavel-II** [Stellarium/stellarium#2116] [Q&A] [answered]
Got this
`
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    USE_PLUGIN_COMPASSMARKS
    WITH_QTWEBENGINE
`
with cmake options:

`cmake ../.. -DENABLE_NLS=OFF -DENABLE_GPS=OFF -DENABLE_LIBGPS=OFF -DENABLE_MEDIA=OFF -DENABLE_SCRIPTING=OFF -DUSE_PLUGIN_ANGLEMEASURE=OFF -DUSE_PLUGIN_COMPASSMARKS=OFF -DUSE_PLUGIN_SATELLITES=OFF -DUSE_PLUGIN_TELESCOPECONTROL=OFF -DUSE_PLUGIN_OCULARS=OFF -DUSE_PLUGIN_TEXTUSERINTERFACE=OFF -DUSE_PLUGIN_SOLARSYSTEMEDITOR=OFF -DUSE_PLUGIN_SUPERNOVAE=OFF -DUSE_PLUGIN_QUASARS=OFF -DUSE_PLUGIN_PULSARS=OFF -DUSE_PLUGIN_EXOPLANETS=OFF -DUSE_PLUGIN_OBSERVABILITY=OFF -DUSE_PLUGIN_ARCHAEOLINES=OFF -DUSE_PLUGIN_EQUATIONOFTIME=OFF -DUSE_PLUGIN_METEORSHOWERS=OFF -DUSE_PLUGIN_NAVSTARS=OFF -DUSE_PLUGIN_NOVAE=OFF -DUSE_PLUGIN_ONLINEQUERIES=OFF -DUSE_PLUGIN_POINTERCOORDINATES=OFF -DUSE_PLUGIN_SCENERY3D=OFF -DUSE_PLUGIN_REMOTECONTROL=OFF -DUSE_PLUGIN_REMOTESYNC=OFF -DWITH_QTWEBENGINE=OFF`

https://github.com/Stellarium/stellarium/blob/master/BUILDING.md 
Supported CMake parameters
neet to update?

**Comment by gzotti** [comment, marked as answer]
You have built earlier versions, and those two options are still stored in your CMakeCache.txt file. You can delete those entries from this file. Else the warning is harmless.

**Reply by alex-w** [reply]
See updated version of [instructions](https://github.com/Stellarium/stellarium/blob/master/BUILDING.md)
